{"uuid": "bc9c24e5-3905-4491-b04e-47d8e7daea64", "children": ["1ab6470e-e62a-44a2-b0dc-f7e551c74e55", "85c6ec7b-2409-4164-991c-de6d07a968e4", "1b20ae1a-d041-4745-b58a-f14a0c2a74f3", "0af71c78-c1c9-4983-897b-f4193b86c77e", "686f94a0-88fe-42da-82c4-29846bc2d738"], "befores": [{"name": "_Class__pytest_setup_class", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.WebDriverException: Message: Can not connect to the Service chromedriver\n\n", "trace": "  File \"/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/pluggy/callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"/Users/yangzhiqi/PycharmProjects/Selenium/venv/lib/python3.9/site-packages/_pytest/fixtures.py\", line 964, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/Users/yangzhiqi/PycharmProjects/Selenium/venv/lib/python3.9/site-packages/_pytest/fixtures.py\", line 788, in call_fixture_func\n    res = next(it)\n  File \"/Users/yangzhiqi/PycharmProjects/Selenium/venv/lib/python3.9/site-packages/_pytest/python.py\", line 701, in xunit_setup_class_fixture\n    _call_with_optional_argument(func, self.obj)\n  File \"/Users/yangzhiqi/PycharmProjects/Selenium/venv/lib/python3.9/site-packages/_pytest/python.py\", line 632, in _call_with_optional_argument\n    func(arg)\n  File \"/Users/yangzhiqi/PycharmProjects/Selenium/TestCase/test_vue.py\", line 23, in setup_class\n    self.driver = webdriver.Chrome()\n  File \"/Users/yangzhiqi/PycharmProjects/Selenium/venv/lib/python3.9/site-packages/selenium/webdriver/chrome/webdriver.py\", line 70, in __init__\n    super(WebDriver, self).__init__(DesiredCapabilities.CHROME['browserName'], \"goog\",\n  File \"/Users/yangzhiqi/PycharmProjects/Selenium/venv/lib/python3.9/site-packages/selenium/webdriver/chromium/webdriver.py\", line 90, in __init__\n    self.service.start()\n  File \"/Users/yangzhiqi/PycharmProjects/Selenium/venv/lib/python3.9/site-packages/selenium/webdriver/common/service.py\", line 105, in start\n    raise WebDriverException(\"Can not connect to the Service %s\" % self.path)\n"}, "start": 1647331802738, "stop": 1647331833334}], "start": 1647331802738, "stop": 1647331833483}